cpa/policy.go

```diff
@@ -15,13 +16,28 @@ type Policy struct {
 
 // Eval will run native OPA against your document, input, and evaluate the query.
```

Comment:
the function comment needs updating to reflect what new arguments it is accepting now